### DISCUSSION TITLE
Supabase database vs Aurora Serverless

### DISCUSSION THREAD

**Discussion by DregondRahl** [supabase/supabase#1313] [Questions]
What's the difference between Supabase PostgresSQL vs Aurora Serverless (postgres flavor)?
Is storage capacity for postgres the main limiter with Superbase ?

Just trying to get an idea what options are there with stitching together a fully serverless solution.

**Comment by dbredvick** [comment]
There is a [pay-as-you-go pricing tier](https://supabase.io/pricing) for Supabase similar to Aurora Serverless. I'm not 100% certain how it's implemented under the hood though, so I can't speak to its elasticity. I'd get in touch with support if you have scale concerns there. (email: support@supabase.io)

For Free Tier the limit is 2GB, for Pro Tier it is 200GB. There are no limits on Pay as you Go.
